scripts/scripts.js

```diff
@@ -305,6 +305,16 @@ async function detectSidebar(main) {
   }
 }
 
+export function debounce(func, delay) {
```

Comment:
Why do we need the debounce function? 🤔

Reply:
to hold back from searching for a small amount (300ms) while the user is typing I thought. superfluous?

Reply:
I had not noticed the `clearTimeout` call, so I thought it still calls the function 3 times if the user types 3 new values. Cool stuff! let's keep it 👍